errai-ui/src/main/java/org/jboss/errai/ui/rebind/TranslationServiceGenerator.java

```diff
@@ -261,7 +260,7 @@ private void addUrlsFromBundleAnnotations(final Collection<MetaClass> bundleAnno
         final URL classpathElement;
         final String pathRoot = getPathRoot(bundleClass, resource);
         try {
-          String urlString = new File(pathRoot).toURI().toURL().toString();
+          String urlString = new URI(pathRoot).toURL().toString();

```

Comment:
Skip passing ```URL``` through ```File``` as ```file:/....``` is interpreted by ```toURI()``` as not absolute and hence the working folder path is appended to what ```File``` thinks is a relative path.